torch/csrc/Stream.cpp

```diff
@@ -53,7 +54,7 @@ PyObject* THPStream_Wrap(const c10::Stream& stream) {
 
   THPStream* self = (THPStream*)ptr.get();
   self->stream_id = stream.id();
-  self->device_index = stream.device_index();
+  self->device_index = static_cast<int64_t>(stream.device_index()); // NOLINT
```

Comment:
NOLINT with which error code?

Reply:
Clang-tidy says int8_t can't be directly casted to int64_t